subprojects/ide-native/src/main/java/org/gradle/ide/xcode/plugins/XcodePlugin.java

```diff
@@ -215,11 +215,11 @@ public void execute(XCTestConventionPlugin plugin) {
     private void configureXcodeForXCTest(final Project project, PBXTarget.ProductType productType) {
         SwiftXCTestSuite component = project.getExtensions().getByType(SwiftXCTestSuite.class);
         FileCollection sources = component.getSwiftSource();
-        xcode.getProject().getSources().from(sources);
+        xcode.getProject().getGroups().getTests().from(sources);
```

Comment:
What about test resources?

Reply:
The test resource, for now, is only `bundleDebug.getInformationFile()`.